packages/omgidl-parser/src/parseIDLToAST.test.ts

```diff
@@ -1918,4 +1918,20 @@ module idl_parser {
     };`;
     expect(() => parseIDLToAST(msgDef)).toThrow(/unexpected RCBR token: "}"/i);
   });
+  it("can parse hexadecimal literals", () => {
+    const msgDef = `
+      const short SHORT_CONSTANT = 0x0C;
```

Comment:
would be good to also test the capital X hex digit here to make sure javascript can parse it correctly as well.